package.json

```diff
@@ -50,8 +50,8 @@
     "element-closest": "^3.0.1",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
-    "eslint": "^5.16.0",
-    "eslint-plugin-shopify": "30.0.1",
+    "eslint": "6.5.1",
```

Comment:
In the end I guess it doesn't matter much but I don't understand hardcoding an exact version as the dependency.